url-sampler/build.gradle.kts

```
@@ -0,0 +1,26 @@
+plugins {
+  id("com.github.johnrengelman.shadow")
+
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+description = "Sampler which makes its decision based on http.url semantic attribute"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  implementation("io.opentelemetry:opentelemetry-sdk")
```

```suggestion
  api("io.opentelemetry:opentelemetry-sdk")
```

Since we extend `Sampler` a public API